.gitignore

```diff
@@ -0,0 +1,3 @@
+/.quarto/
```

Comment:
I would take out this file

Reply:
i.e. do not include `.gitignore` in this pr

Reply:
I put ignored the compiled file directories, is there a reason you dont want `.gitignore`